.ci/script.sh

```diff
@@ -18,7 +18,7 @@ if [ "$TRAVIS_CMAKE_GENERATOR" = "Visual Studio 15 2017" ] ; then
     cmake -G"$TRAVIS_CMAKE_GENERATOR" \
           -A"${TRAVIS_CMAKE_ARCHITECTURE}" \
           -DCMAKE_INSTALL_PREFIX="$INSTALL_PREFIX" \
-          -DBUILD_TESTING:BOOL=ON .. \
+          -DBUILD_TESTING:BOOL=${BUILD_TESTING:-0} .. \
```

Comment:
My ignorance: what is `${BUILD_TESTING:-0}` (in particular `:-0`?).

Reply:
It allows using `set -u` and having a default value of the variable if it is not defined. I like it because I can avoid to define `BUILD_TESTING=0` in the travis build matrix and just define it when it is on (the online interface is cleaner).